README.md

```diff
@@ -1,24 +1,66 @@
-# basic-storage-integration
+<p>
+<a href="#start"><img height="30rem" src="https://raw.githubusercontent.com/arcana-network/branding/main/an_logo_light_temp.png"/></a>
+</p>
+<p>
+<a title="MIT License" href="https://github.com/arcana-network/license/blob/main/LICENSE.md"><img src="https://img.shields.io/badge/license-MIT-blue"/></a>
+<a title="Beta release" href="https://github.com/arcana-network/basic-storage-wallet-integration/releases"><img src="https://img.shields.io/github/v/release/arcana-network/basic-storage-wallet-integration?style=flat-square&color=28A745"/></a>
+<a title="Twitter" href="https://twitter.com/ArcanaNetwork"><img alt="Twitter URL" src="https://img.shields.io/twitter/url?style=social&url=https%3A%2F%2Ftwitter.com%2FArcanaNetwork"/></a>
+</p><p id="start" align="center">
+</p>
+
+# Sample Code: Basic Storage and Auth SDK integration
+
+The repository `basic-storage-wallet-integration` contains sample code to demonstrate how you can integrate with Arcana [Auth](https://github.com/arcana-network/auth) and [Storage](https://github.com/arcana-network/storage) SDKs.
+
+It does not have any code for invoking other features and functions of these integrated SDKs.  Check out Arcana [Sample Demo App](https://github.com/arcana-network/demo-app) repository for a full fledged sample dApp that integrates with Arcana SDKs and plugs in Auth and Storage functionalities.
+
+## ⚙️ Installation
+
+### Deploy Software Dependencies
 
-## Project setup
 ```
 yarn install
 ```
 
-### Compiles and hot-reloads for development
+### Run Development Server
+
+To re-compile and hot-reload during development, use:
+
 ```
 yarn serve
 ```
 
-### Compiles and minifies for production
+### Production Build
+
+To compile and minify for production build, use:
 ```
 yarn build
 ```
 
-### Lints and fixes files
+### Lint and Fix Files
+
 ```
 yarn lint
 ```
 
-### Customize configuration
-See [Configuration Reference](https://cli.vuejs.org/config/).
+### Customize Configuration
+
+See [Configuration Reference](https://cli.vuejs.org/config/) for customization details.
+
+## 📚 Usage
+
+You can utilize the source code in this repository for getting started with Arcana Network SDKs. Use these as stand alone basic applications to integrate with the latest Auth and Storage SDK.  You may plug in additional functionality and try out various features of the SDKs.
+
+See [Arcana Documentation](https://docs.beta.arcana.network/) for user guides and SDK Reference Guides.  
+
+To get started with Arcana SDKs quickly, you can also check out other [code samples and templates](https://docs.beta.arcana.network/docs/overview_cs) in the `Developer Resources` section of the documentation.
+
+## 💡 Support
+
+For any queries regarding this repository usage, contact [Arcana support team](mailto:support@arcana.network).
```

Comment:
Shouldn't there be a “the” here? (the Arcana support team)
Plus I think it also needs title casing.